msteams-platform/messaging-extensions/high-quality-message-extension.md

```diff
@@ -34,11 +34,11 @@ The requirements for building message extension plugins for Copilot for Microsof
 
 ## Define descriptions
 
-A good description offers a clear and concise summary of the app’s features and allows Copilot for Microsoft 365 to efficiently discover and execute search operations. When a user enters the app name along with a verb, for example, **Find Contoso tickets**, the message extension plugin must be invoked from Microsoft 365 Chat.
+A good description offers a clear and concise summary of the app’s features and allows Copilot for Microsoft 365 to efficiently discover and execute search operations. When a user enters the app name along with a verb, for example, **Find Contoso tickets**, the message extension plugin must be invoked from Microsoft 365 Chat (M365 Chat).
```

Comment:
How about:

must be invoked from Copilot for Microsoft 365 (M365 Chat)

Reply:
Updated.